.github/workflows/ci-pipeline.yml

```diff
@@ -365,16 +365,16 @@ jobs:
     - name: Setup Coq
       run: echo "::add-path::/home/coq/.opam/$COMPILER/bin"
     - name: Build Coq base library
-      run: ./tool/compile-coq.sh base
+      run: ./tool/compile-coq.sh base/coq
     - name: Copy compiled base library to build directory
       run: |
-        mkdir -p ./build/base
+        mkdir -p ./build/base/coq
         cp --parents $(find base -name "*.vo" -or -name "*.glob") ./build
```

Comment:
Even though the Agda base library should not contain any `.vo` or `.glob` files, I'd explicitly search in `base/coq` only.